app/deliver_grant_funding/templates/deliver_grant_funding/grant_list.html

```diff
@@ -32,7 +32,7 @@ <h1 class="govuk-heading-l">Grants</h1>
         {%
           set table_rows.items = table_rows.items + [
               [{"html": grant_detail_link},
-               {"text": grant.ggis_number or ''},
+               {"text": grant.ggis_number or 'Not known'},
```

Comment:
As per Kate's slack message, this should be

```suggestion
               {"text": grant.ggis_number or 'Not provided'},
```